src/handlers/expenditures/helpers/getExpenditure.ts

```diff
@@ -34,15 +29,11 @@ export const getExpenditureFromDB = async (
 /**
  * Utility checking if `getExpenditure` function exists on the given colony client
  */
-type GetExpenditureSupportedColonyClient = Exclude<
-  AnyColonyClient,
-  ColonyClientV1 | ColonyClientV2 | ColonyClientV3
->;
+type GetExpenditureSupportedColonyClient = AnyColonyClient;
 const isGetExpenditureSupported = (
   colonyClient: AnyColonyClient,
 ): colonyClient is GetExpenditureSupportedColonyClient =>
-  (colonyClient as GetExpenditureSupportedColonyClient).getExpenditure !==
-  undefined;
+  colonyClient.getExpenditure !== undefined;
```

Comment:
You can get rid of the type and the `colonyClient is GetExpenditureSupportedColonyClient`, it doesn't really do anything following colony-js update